velox/functions/lib/JsonArrayLength.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/functions/Macros.h"
+#include "velox/functions/prestosql/json/JsonPathTokenizer.h"
+#include "velox/functions/prestosql/json/SIMDJsonExtractor.h"
+#include "velox/functions/prestosql/json/SIMDJsonWrapper.h"
+#include "velox/functions/prestosql/types/JsonType.h"
+
+namespace facebook::velox::functions {
+
+/// json_array_length(jsonString) -> length
+///
+/// Returns the array length of json if a valid json string is given.
+/// Returns null if the input json string is invalid.
+/// It is used both in SparkSQL and Presto, See documentation at
```

Comment:
> It is used both in SparkSQL and Presto

nit: perhaps remove this description and keep the doc links only, as the implementations in the 'velox/functions/lib' are meant to be used by both Presto and Spark.

Reply:
done